lib/features/mailbox_creator/presentation/mailbox_creator_controller.dart

```diff
@@ -29,6 +29,7 @@ class MailboxCreatorController extends BaseController {
 
   final selectedMailbox = Rxn<PresentationMailbox>();
   final newNameMailbox = Rxn<String>();
+  final _createdMailbox = RxBool(false);
```

Comment:
Use the `Obx` variable only when you want to listen for changes to UI updates. So please remove it.